ClothesApp/Application/Services/IBrandService.cs

```diff
@@ -0,0 +1,16 @@
+using Application.Dtos;
+
+namespace Application.Services;
+
+public interface IBrandService
+{
+    Task<BrandDto> DeleteBrandById(long id);
```

Comment:
All 5 methods are async, but this one doesn't have this word, other 4 have, this is very confusing, and that is why I don't like async word